pkg/common/ingress/annotations/dnsresolvers/main.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dnsresolvers
+
+import (
+	"github.com/golang/glog"
+	"github.com/jcmoraisjr/haproxy-ingress/pkg/common/ingress/annotations/parser"
+	"github.com/jcmoraisjr/haproxy-ingress/pkg/common/ingress/resolver"
+	extensions "k8s.io/api/extensions/v1beta1"
+	"reflect"
+	"strings"
+)
+
+const (
+	DNSResolversAnn = "ingress.kubernetes.io/dns-resolvers"
+	UseResolverAnn  = "ingress.kubernetes.io/use-resolver"
+)
```

Comment:
const is a local declaration, start the identifiers with lowercase instead

Reply:
sure, np